### PR TITLE
[integration-tests] Expanded upon basic integration test

### PR DESCRIPTION
**Pull request by ebebbington**

### CODE REVIEW

**Comment by ebebbington** [comment]
FYI With the above commits, i've had to adjust how rhum runs the test cases because the output was super wierd - but now the inetegration tests still fail, but there is no actual and expectations in the output... so the test is failing but not saying where - i assume it's a problem with `\u0008`

**Comment by ebebbington** [comment]
Thanks to @crookse who managed to solve the problem of asserting to stdout!!!!!

File: deps.ts [hunk 1]
```diff
@@ -3,3 +3,5 @@ export { ServerRequest } from "https://deno.land/std@v0.57.0/http/server.ts";
 export { BufReader } from "https://deno.land/std@v0.57.0/io/bufio.ts";
```
Comment (crookse) [inline deps.ts:3]: we should bump to v0.58.0 since we're using deno 1.1.1

Comment (crookse) [reply deps.ts:3]: nvm the other PR includes this

**Pull request merged**